sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/json/JSONOptions.scala

```diff
@@ -34,35 +34,42 @@ private[sql] class JSONOptions(
     @transient private val parameters: Map[String, String])
```

Comment:
Oh, sure! You mean `require(parameters.isInstanceOf[CaseInsensitiveMap])`, right?